docs/installing/index.md

```diff
@@ -6,61 +6,96 @@ title: Installing
 # Installing Alaveteli
 
 <p class="lead">
-  There are a number of ways to install Alaveteli.
-  We've made an Amazon Machine Image (AMI) so you can quickly deploy on
-  Amazon EC2 (handy if you just want to evaluate it, for example).
-  If you prefer to use your own server, there's an installation script
-  which does most of the work for you, or you can follow the manual
-  installation instructions.
+  There are a number of ways to install Alaveteli, but we recommend
+  you begin with the Vagrant installation to get a development
+  site up and running.
 </p>
 
 ## Before you start
 
-This is important: you need to decide if you are installing Alaveteli for
+This is important: there is a difference between installing Alaveteli for
 <a href="{{ page.baseurl }}/docs/glossary/#development" class="glossary__link">development</a> or
 <a href="{{ page.baseurl }}/docs/glossary/#production" class="glossary__link">production</a>.
 
-A **development** site is one where you're going to change, customise, and
-perhaps experiment while you get it up and running. You should always do this
-first. In this environment you can see debug messages, and you don't need to
+You *always* start with a **development** site. This is one you're going to change, customise, and
+perhaps experiment while you get it up and running. Every Alaveteli installation requires some customisation (it's designed this way!)
+including creating your own 
+<a href="{{ page.baseurl }}/docs/glossary/#theme" class="glossary__link">theme</a>. In this environment you can see debug messages, and you don't need to
 worry too much about the efficiency and performance of the site (because it's
 not really getting lots of traffic).
 
 A **production** site is different: you want your production site to run as
 efficiently as possible, so things like caching are swiched on, and debug
```

Comment:
for extra credit, could you fix the spelling of "swi***t***ched on" while you're here? ta :smile: